__tests__/app.test.js

```diff
@@ -0,0 +1,48 @@
+const { expect } = require("@jest/globals");
+const request = require("supertest");
+const app = require("../app/app");
+const db  = require("../db/connection");
+const dataTest = require("../db/data/test-data/index")
+const seed = require("../db/seeds/seed")
+
+beforeEach(() => {
+    return seed(dataTest);
+})
+
+afterAll(() => {
+    db.end();
```

Comment:
Good stuff putting in the after all and beforeeach just db.end needs returning here 👍